functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/failed/SurefireReportHandlingUtils.java

```diff
@@ -0,0 +1,36 @@
+package org.arquillian.smart.testing.ftest.failed;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.arquillian.smart.testing.ftest.testbed.project.Project;
+
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
+import static org.arquillian.smart.testing.ftest.testbed.project.ProjectBuilder.IN_PROJECT_DIR;
+import static org.arquillian.smart.testing.ftest.testbed.project.ProjectBuilder.TEST_REPORT_PREFIX;
+
+public class SurefireReportHandlingUtils {
+
+    private static final String AFFECTED_MODULE_PATH = "container/impl-base";
```

Comment:
I would avoid such a constraint - it locks us with the concrete project to be used for tests. Do we really need this?